### PR TITLE
feat: [FFM-3327]: Standardising button variations in the environments pages

### PR DESCRIPTION
**Pull request by laura-mannus**
##### Summary:

**Environments listing page**

“+ New Environment” - change to use primary variation

**Environment detail page**

_Empty state_

“Create SDK Key” - change to use primary variation

_Non-empty state_

“Create SDK Key” - change to use primary variation

- Using relative css path
- Removed css classes that were not needed in layout containers
- Dropping unnecessary Container

##### Jira Links:

https://harness.atlassian.net/browse/FFM-3327

##### Screenshots:

image

image

image

image

You can use the following comments to re-trigger PR Checks

- Jest: `retrigger jest`
- Prettier: `retrigger prettier`
- Type Check: `retrigger typecheck`
- ESLint: `retrigger eslint`
- Standards: `retrigger standards`
- Build: `retrigger build`
- Title Check: `retrigger titlecheck`
- Coverage: `retrigger coverage`
- Rebase: `trigger rebase`
- Cypress: `trigger cypress`
- Fix Prettier - `fix prettier`

### CODE REVIEW

**Comment by laura-mannus** [comment]
retrigger jest

**Comment by laura-mannus** [comment]
retrigger jest

**Comment by laura-mannus** [comment]
retrigger jest

**Comment by laura-mannus** [comment]
retrigger jest

**Review by chrisblakely01 — CHANGES_REQUESTED** [review]
We probs want to rollback the `75-cd` changes as its not our module/out of scope for this ticket

File: src/modules/75-cf/components/AddKeyDialog/AddKeyDialog.tsx [hunk 1]
```diff
@@ -155,6 +155,7 @@ const AddKeyDialog: React.FC<Props> = ({ disabled, primary, environment, onCreat
   return (
     <RbacButton
```
Comment (crystal-fong) [inline src/modules/75-cf/components/AddKeyDialog/AddKeyDialog.tsx:156]: this won't let me comment on the exact line that needs changed but line 165

`text={getString('cf.environments.apiKeys.addKeyTitle')}`

File: src/modules/75-cd/components/Environments/EnvironmentList/EnvironmentsList.tsx [hunk 2]
```diff
@@ -7,7 +7,7 @@
 
 import React, { useEffect, useMemo, useState } from 'react'
 import cx from 'classnames'
-import { Pagination, Layout, Text, Container, Heading, TableV2 } from '@wings-software/uicore'
+import { Pagination, Layout, Text, Container, Heading, TableV2, ButtonVariation } from '@wings-software/uicore'
```
Comment (crystal-fong) [inline src/modules/75-cd/components/Environments/EnvironmentList/EnvironmentsList.tsx:10]: if you're pushing changes anyway :) 

```suggestion
import { Pagination, Layout, Text, Container, Heading, TableV2, ButtonVariation } from '@harness/uicore'
```

Comment (laura-mannus) [reply src/modules/75-cd/components/Environments/EnvironmentList/EnvironmentsList.tsx]: Unfortunately can't fix this as its not our module, have removed the change altogether

File: src/modules/75-cf/components/AddKeyDialog/AddKeyDialog.tsx (hunk 1 above)
Comment (laura-mannus) [reply src/modules/75-cf/components/AddKeyDialog/AddKeyDialog.tsx:156]: Thank you! Have changed key title to '+ New SDK Key' 
image

File: src/modules/75-cf/pages/environments/EnvironmentsPage.tsx [hunk 3]
```diff
@@ -52,17 +51,14 @@ export const NameCell = withEnvironment(({ environment }) => {
...
     >
-      <Layout.Vertical className={css.calcMaxWidth} spacing="xsmall">
+      <Layout.Vertical spacing="xsmall">
         <Text style={{ color: Color.BLACK, font: FontVariation.BODY2 }}>{environment.name}</Text>
 
         {environment.description && (
-          <Container>
-            <Text style={{ font: FontVariation.SMALL, color: Color.GREY_600 }}>{environment.description}</Text>
-          </Container>
+          <Text style={{ font: FontVariation.SMALL, color: Color.GREY_600 }}>{environment.description}</Text>
```
Comment (knagurski) [inline src/modules/75-cf/pages/environments/EnvironmentsPage.tsx:61]: Don't know how I didn't spot this before, but for this (and the one above), we need to use the `font` prop, not the `style` prop for variations and colour
```suggestion
          <Text font={{ variation: FontVariation.SMALL, color: Color.GREY_600 }}>{environment.description}</Text>
```

Comment (laura-mannus) [reply src/modules/75-cf/pages/environments/EnvironmentsPage.tsx:61]: Don't think color is a prop of font?

Comment (crystal-fong) [reply src/modules/75-cf/pages/environments/EnvironmentsPage.tsx:61]:
```suggestion
          <Text color={Color.GREY_600} font={{ font: FontVariation.SMALL }}>{environment.description}</Text>
```

Comment (laura-mannus) [reply src/modules/75-cf/pages/environments/EnvironmentsPage.tsx]: Lovely thank you have changed

**Review by knagurski — APPROVED** [review]
Noice

**Pull request merged**